rflx/model.py

```diff
@@ -1093,7 +1042,7 @@ def resolve_type(expr: Expr) -> Optional[TypeExpr]:
 
         def invalid_relation(left: TypeExpr, right: TypeExpr) -> bool:
             return (
-                (isinstance(left, Opaque) and not isinstance(right, (Opaque, Aggregate)))
+                (isinstance(left, Opaque) and not isinstance(right, (Opaque, Aggregate, String)))
```

Comment:
Shouldn't testing for `Aggregate` be enough now when `String` is a child class?

Reply:
You are right. Fixed by 73788a0.